dev-tools/mage/pytest.go

```diff
@@ -32,6 +32,14 @@ import (
 	"github.com/magefile/mage/sh"
 )
 
+// WINDOWS USERS:
+// The python installer does not create a python3 alias like it does on other
+// platforms. So do verify the version with python.exe --version.
+//
+// Setting up a python virtual environment on a network drive does not work
+// well. So if this applies to your development environment set PYTHON_EXE
```

Comment:
```suggestion
// well. So if this applies to your development environment set PYTHON_ENV
```

Reply:
Fixed in https://github.com/elastic/beats/pull/16364, where I am also adding a changelog entry for this.